modules/light-clients/07-tendermint/types/client_state.go

```diff
@@ -499,18 +507,29 @@ func (cs ClientState) VerifyNextSequenceRecv(
 
 // verifyDelayPeriodPassed will ensure that at least delayPeriod amount of time has passed since consensus state was submitted
 // before allowing verification to continue.
-func verifyDelayPeriodPassed(store sdk.KVStore, proofHeight exported.Height, currentTimestamp, delayPeriod uint64) error {
-	// check that executing chain's timestamp has passed consensusState's processed time + delay period
+func verifyDelayPeriodPassed(store sdk.KVStore, proofHeight, currentHeight exported.Height, currentTimestamp, delayTimePeriod, delayBlockPeriod uint64) error {
+	// check that executing chain's timestamp has passed consensusState's processed time + delay time period
 	processedTime, ok := GetProcessedTime(store, proofHeight)
 	if !ok {
 		return sdkerrors.Wrapf(ErrProcessedTimeNotFound, "processed time not found for height: %s", proofHeight)
 	}
-	validTime := processedTime + delayPeriod
-	// NOTE: delay period is inclusive, so if currentTimestamp is validTime, then we return no error
+	validTime := processedTime + delayTimePeriod
+	// NOTE: delay time period is inclusive, so if currentTimestamp is validTime, then we return no error
 	if validTime > currentTimestamp {
```

Comment:
nit: `currentTimestamp < validTime` to match if statement below. I think it might be easier to read